application/utils/harvester/diff_retriever.py

```
@@ -0,0 +1,76 @@
+
+            ValueError:
+                If the diff exceeds the configured size limit.
+        """
+        logger.info(
+            "Retrieving diff between %s and %s",
+            base_commit,
+            target_commit,
+        )
+
+        try:
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)